src/utils/users.ts

```diff
@@ -0,0 +1,70 @@
+import { User } from "../models/user";
+
+type LocalStorageKey = "users";
```

Comment:
As mentioned above, we should add "state" to this type, as we also store app state in local storage (not just users).